src/main/java/com/epam/rd/autotasks/springemployeecatalog/constants/EmployeeEnum.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.rd.autotasks.springemployeecatalog.constants;
+
+public enum EmployeeEnum {
+    employeeId("ID"),
+    id("ID_e"),
+    employeeFirstname("FIRSTNAME"),
+    firstname("FIRSTNAME_e"),
+    managerFirstname("FIRSTNAME_m"),
+    employeeLastname("LASTNAME"),
+    lastname("LASTNAME_e"),
+    managerLastname("LASTNAME_m"),
+    employeeMiddlename("MIDDLENAME"),
+    middlename("MIDDLENAME_e"),
+    managerMiddlename("MIDDLENAME_m"),
+    employeeHiredate("HIREDATE"),
+    hiredate("HIREDATE_e"),
+    managerHiredate("HIREDATE_m"),
+    employeePosition("POSITION"),
+    position("POSITION_e"),
+    managerPosition("POSITION_m"),
+    employeeSalary("SALARY"),
+    salary("SALARY_e"),
+    managerSalary("SALARY_m"),
+    employeeManager("MANAGER"),
+    manager("MANAGER_e"),
+    department("department"),
+    name("NAME"),
+    location("LOCATION");
+
+    private final String column;
+
+    EmployeeEnum(String columnName) {
+        column = columnName;
+    }
+
+    public String toString() {
+        return this.column;
+    }
```

Comment:
Please do not override this method. It would be better to create getter and use it
public String getColumnName() {
        return column;
    }

Also it is not not column, it is column name